modules/dev-utils/src/main/java/org/apache/ignite/development/utils/IgniteWalConverter.java

```diff
@@ -53,8 +74,12 @@ public static void main(String[] args) throws Exception {
         H2ExtrasInnerIO.register();
         H2ExtrasLeafIO.register();
 
-        boolean printRecords = IgniteSystemProperties.getBoolean("PRINT_RECORDS", false); //TODO read them from argumetns
-        boolean printStat = IgniteSystemProperties.getBoolean("PRINT_STAT", true); //TODO read them from argumetns
+        boolean printRecords = getBoolean(PRINT_RECORDS, false); //TODO read them from argumetns
+        boolean printStat = getBoolean("PRINT_STAT", true); //TODO read them from argumetns
+        ProcessSensitiveData sensitiveData = getEnum(SENSITIVE_DATA, SHOW); //TODO read them from argumetns
```

Comment:
Please create a ticket for this TODOs and link it to yours.

Reply:
https://ggsystems.atlassian.net/browse/GG-28576